### ISSUE TITLE
date format not correct

### ISSUE THREAD

**Issue by cadonovan** [kevinpapst/kimai2#2583]
**Describe the bug**
I have installed Kimai from Softaculous and all is working well except the settings for date formats. I chose Language = English, Timezone = Australia / Perth for both the customer defaults and the user defaults.

In Australia, the date format should be dd/mm/yyyy. (Also note that 'en' language also uses the same format as Australia and should NOT be yyyy-mm-dd). However, Kimai is using yyyy-mm-dd formats for every calendar, input screen, report and invoice. I cannot seem to correct it even though I have changed the 'en' settings in the config files (config/packages/kimai.yaml) which were incorrect.

It now is the following :
en:
            date_time_type: 'dd-MM-yyyy HH:mm'
            date_type: 'dd-MM-yyyy'
            date: 'd-m-Y'
            date_time: 'd-m H:i'
            duration: '%%h:%%m h'

However, it is not showing the correct date format within Kimai or in reports and invoices. I have tried adding new users and customers but it does not show the correct date format.

Regards

Craig

**Additional context**
 - Kimai version: 1.14.1 stable
 - PHP version: 7.3.27
 - Device: Windows 10 on Microsoft Surface Pro 6
 - Browser: Vivaldi

**Comment by kevinpapst** [comment]
Did you read the documentation about configs? Did you clear the cache?

**Comment by cadonovan** [comment]
> Did you read the documentation about configs? Did you clear the cache?

HI Kevin
I had trouble trying to clear the cache previously but I have managed to do so now using the command line "bin/console kimai:reload --env=prod" in the cPanel Terminal screen. I needed to navigate inside the kimai install folder for it to work (which I did not realise previously).

All working fine now.

However, you might want to suggest a change to the default "en" date formats or perhaps create an "au" date format, so that it displays dd/mm/yyyy.

Thanks
Craig
